reporting-engine-worker/pom.xml

```diff
@@ -97,6 +97,15 @@
 										<prefix>/opt/${packageName}/</prefix>
 									</mapper>
 								</data>
+								<data>
+									<src>${basedir}/scripts</src>
+									<dst>scripts</dst>
+									<type>directory</type>
+									<mapper>
+										<type>perm</type>
+										<prefix>/opt/${packageName}/scripts</prefix>
+									</mapper>
+								</data>
```

Comment:
This is a great way to include the scripts we need.   I'm happy that they're going to be part of the source code under version control and deployed in sync, and obviously the OS needs to see them _outside_ of the Jar file so this is the right approach.    I'm interested to see this working.

Reply:
@pgwilliams Yes, I am interested too, can only test by deploying.